src/module/index.js

```diff
@@ -3,7 +3,7 @@ import { GlobalVue } from '../index';
 import * as types from './mutation-types';
 import * as actions from './actions';
 
-const mutations = {
+const mutations = Object.seal({
```

Comment:
Why not even freeze?

Reply:
probably freeze here makes sense assuming this object will be never touched by anyone